modAionImpl/test/org/aion/zero/impl/PendingStateTest.java

```diff
@@ -0,0 +1,59 @@
+package org.aion.zero.impl;
+
+import java.math.BigInteger;
+import org.aion.base.type.Address;
+import org.aion.crypto.ECKey;
+import org.aion.mcf.blockchain.TxResponse;
+import static org.junit.Assert.*;
+import org.aion.zero.impl.config.CfgAion;
+import org.aion.zero.types.AionTransaction;
+import org.junit.Test;
+
+public class PendingStateTest {
+
+    @Test
+    public void TestAddPendingTransaction() {
```

Comment:
Looks good!  Thanks for adding the test.  Minor suggestion -- I would make two test cases; one for the successful case and one for the invalid nrg price case

Reply:
Done! :)